Source/ChocolateyGui/Services/ChocolateyService.cs

```diff
@@ -446,18 +434,17 @@ public Task<bool> IsElevated()
 
         public async Task<bool> RemoveSource(string id)
         {
-            var operationContext = OperationContext.Current;
             using (await Lock.WriteLockAsync())
             {
-                var chocoConfig = await GetConfigFile(operationContext);
+                var chocoConfig = await GetConfigFile();

```

Comment:
Maybe you already reverted this, but I don't think these lines are stricktly necessary? Unless you had a reason to add them back?